database_management_bot.py

```diff
@@ -0,0 +1,2235 @@
+    logger.info("%s: File warning", user.first_name)
+
+    update.message.reply_text(
+        text='Будь ласка, використовуйте *фотографію*, а не файл\.',
+        parse_mode='MarkdownV2',
+    )
+
+
+@under_maintenance
+def main_keyboard_handler(update: Update, context: CallbackContext) -> None:
+    """
+    The main_keyboard_handler function is a callback function that is called
+    in order to show MAIN_REPLY_KEYBOARD to the user.
+
+    :param update: Update: Pass the current update to the function
+    :param context: CallbackContext: Pass information between different parts of the program
+    :return: None
+    """
+    user = update.message.from_user
+    logger.info("%s: %s. Returning to the main menu", user.first_name, update.message.text)
+
+    reply_keyboard = MAIN_REPLY_KEYBOARD
+
+    if update.message.text not in ["Зрозуміло!", "Ні"]:
+        update.message.reply_text(
+            text='☑️ Сканування завершено',
+            reply_markup=ReplyKeyboardMarkup(
+                reply_keyboard,
+                one_time_keyboard=True,
+                resize_keyboard=True,
+                input_field_placeholder='Оберіть опцію',
+            ),
+        )
+    else:
+        update.message.reply_text(
+            text='Гаразд',
+            reply_markup=ReplyKeyboardMarkup(
+                reply_keyboard,
+                one_time_keyboard=True,
+                resize_keyboard=True,
+                input_field_placeholder='Оберіть опцію',
+            ),
+        )
+
+
+@under_maintenance
+def instructions_handler(update: Update, context: CallbackContext) -> ConversationHandler.END:
+    """
+    The instructions_handler function is called whenever the user sends a message that matches
+    the regular expression '^(Інструкції|/help)$'.
+    It is used to send instructions to users. It also returns ConversationHandler.END
+
+
+    :param update: Update: Access the message object
+    :param context: CallbackContext: Send data back to the bot
+    :return: ConversationHandler.END
+    """
+    user = update.message.from_user
+    logger.info("%s: %s", user.first_name, update.message.text)
+
+    reply_keyboard = [['Зрозуміло!']]
+
+    update.message.reply_photo(
+        open('resources/How_to_scan.png', 'rb'),
+        caption='🔍 *Щоб перевірити наявність* штрих\-коду у базі даних \- надішліть мені фото '
+                'пакування, де я можу *чітко* побачити штрихкод\.'
+                '\n\n☝️️ Зверніть увагу, ви можете надсилати *одразу декілька фотографій*\.'
+                '\n\n❗️ Аби *додати* новий медикамент до бази даних, оберіть опцію '
+                '"Додати новий медикамент", або скористайтесь командою */add*'
+                '\n\nТакож, *бот сам запропонує* Вам додати відсутній штрих\-код\.'
+                '\n\n☑️ При додаванні вкажіть назву медикаменту\. *Не використовуйте спеціальних символів* у назві\.'
+                '\n☑️ Далі вкажіть діючу речовину даного препарату\.'
```

Comment:
Sounds better to me. Or "Після цього."
```suggestion
                '\n☑️ Потім вкажіть діючу речовину даного препарату\.'
```

Reply:
Accepted